packages/headless/src/features/facets/facet-set/facet-set-slice.ts

```diff
@@ -79,19 +79,19 @@ export const facetSetReducer = createReducer(
           return;
         }
 
-        const targetValue = facetRequest.currentValues.find(
+        facetRequest.preventAutoSelect = true;
```

Comment:
Since we expect people to use this mechanism to bootstrap their UI with preselected values on load, I guess it makes sense that we would not also allow auto selection from the API in that scenario.

Reply:
Adding a test check that it is set to true when a new value is added would be nice.

Said differently, if this line was moved to the bottom, I don't think any test atm would fail? But it would lead to a change in behavior,